index.html

```diff
@@ -5,8 +5,43 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="/styles/index.css" rel="stylesheet" />
 </head>
 <body>
+    <header>
+        <p id="sky-emoji">☁️ ☁️ ☁️ ☀️ ☁️ ☁️ ☀️ ☁️ ☁️ ☀️ ☁️ ☁️ ☁️</p>
+        <h1 id="weather-report">Weather Report</h1>
+        
+    </header>
+    <main >
+        <ul class="box-container">
```

Comment:
Style: This data isn't actually a list of things, so having it be a `ul` isn't the best semantic choice.

Making this a section would be preferable.